File: INICIANTE/JAVASCRIPT/1164.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift(); };

var N = prompt("")
var soma = 0

for(i=0;i<N;i++){
    var x = parseInt(prompt(""))

    for(k=1;k<=x;k++){
        if(x%k==0){
            soma = soma + k
        }
    }
        y = soma - x
        if(y == x){
            console.log(x + " eh perfeito")
        }else{
            console.log(x + " nao eh perfeito")
    }
    soma = 0
}

File: INICIANTE/JAVA/1164.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int N = sc.nextInt();
        int soma = 0;
        
        for(int i = 0; i < N; i++){
            int x = sc.nextInt();
        
            for(int k = 1; k <= x; k++){
                if(x % k == 0){
                    soma = soma + k;
                }
            }
                int y = soma - x;
                if(y == x){
                    System.out.println(x + " eh perfeito");
                }else{
                    System.out.println(x + " nao eh perfeito");
            }
            soma = 0;
        }
        sc.close();
    }
}
